default.nix

```diff
@@ -3,7 +3,13 @@
 , compiler ? "ghc948"
 
 , # Commit of the Nixpkgs repository that we want to use.
-  nixpkgsVersion ? import nix/nixpkgs-version.nix
+  nixpkgsVersion ? let
+    lock = builtins.fromJSON (builtins.readFile ./flake.lock);
+  in
+  {
+    inherit (lock.nodes.nixpkgs.locked) owner repo rev;
+    tarballHash = lock.nodes.nixpkgs.locked.narHash;
+  }
```

Comment:
Cool, so this means that doing `nix-build --attr postgrestStatic` to get the static bin will keep working as usual.